clang-tidy-review CI fialing on stable branch

```
CMake Error at cmake/FreecivDependencies.cmake:24 (find_package):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" (requested
  version 5.15) with any of the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:75 (include)


-- Configuring incomplete, errors occurred!
Error: Process completed with exit code 1.
```

--- a/.github/workflows/clang-tidy.yaml
+++ b/.github/workflows/clang-tidy.yaml
@@ -4,8 +4,10 @@
 name: clang-tidy-review
 on:
   # We need to run on pull_request_target to gain access to secrets (for the
-  # Codacy token).
+  # Codacy token). Limit to run on master branch.
   pull_request_target: {}
+    branches:
+      - master
 
 concurrency:
   group: ${{ github.repository }}-${{ github.head_ref || github.run_id }}-clang-tidy

--- a/.github/workflows/clang-tidy.yaml
+++ b/.github/workflows/clang-tidy.yaml
@@ -5,7 +5,7 @@ name: clang-tidy-review
 on:
   # We need to run on pull_request_target to gain access to secrets (for the
   # Codacy token). Limit to run on master branch.
-  pull_request_target: {}
+  pull_request_target:
     branches:
       - master
 

